composer.json

```diff
@@ -29,7 +29,7 @@
         "superbalist/laravel-google-cloud-storage": "^2.2"
     },
     "require-dev": {
-        "barryvdh/laravel-ide-helper": "^2.10",
+        "barryvdh/laravel-ide-helper": ">=2.8.0 <2.9.2",
```

Comment:
This change will also require a new / updated composer.lock to be committedd

Reply:
done